package.json

```diff
@@ -95,18 +93,16 @@
     "listr": "^0.13.0",
     "loader-utils": "^1.1.0",
     "lodash": "^4.17.5",
-    "log-symbols": "2.2.0",
+    "log-symbols": "^2.2.0",
     "mkdirp": "^0.5.1",
     "p-each-series": "^1.0.0",
     "p-lazy": "^1.0.0",
     "prettier": "^1.5.3",
-    "recast": "^0.14.4",
```

Comment:
We don't use `recast` inside the code? :O

Reply:
No, a recast object is passed around but it looks like it's created by jscodeshift (which has its own dependency on a different version of recast)

Reply:
Great! Thanks